client/my-sites/plan-features-2023-grid/components/features.tsx

```diff
@@ -1,8 +1,8 @@
 import { getPlanClass, FEATURE_CUSTOM_DOMAIN, isFreePlan } from '@automattic/calypso-products';
+import { useGetWordPressSubdomain } from '@automattic/data-stores/src/domain-suggestions';
```

Comment:
Thinking whether it would be better to have `import { DomainSuggestions } from '@automattic/data-stores';` instead - and reach the query via `DomainSuggestions.useGetWordPressSubdomain`. 

Although, could that imply having the store registered as a side effect? (I think it won't be the case here, as we still need to explicitly call the `register` function, but it might become the case once the store is refactored like the others) If not a concern, then I'd go for `import { DomainSuggestions } from '@automattic/data-stores';`

Reply:
Done via 5061f39b6bc2ab0314dedef4732688dbf8fa692e